### PR TITLE
62729: Add Native Helm Limitation to GitOps

### PR DESCRIPTION
**Pull request by jonquil2002**
SC: https://app.shortcut.com/replicated/story/62729/add-gitops-helm-limitation-to-gitops-topic

Preview: https://deploy-preview-798--replicated-docs.netlify.app/enterprise/gitops-workflow#limitations

### CODE REVIEW

**Comment by paigecalvert** [comment]
Related to issue: https://github.com/replicatedhq/replicated-docs/issues/794

File: docs/enterprise/gitops-workflow.md [hunk 1]
```diff
@@ -14,15 +14,17 @@ If you configure automatic updates for the application, any updates from your ve
 
 You can change your GitOps settings or disable a GitOps workflow at any time from the GitOps tab.
 
-## Limitation
+## Limitations
 
-To enable pushing updates through the GitOps workflow, you must first follow the installation workflow for the application using the admin console or the kots CLI. If the preflight checks pass during installation, then the application is deployed.
+- Native Helm chart installations are not supported by the GitOps workflow.
```
Comment (paigecalvert) [inline docs/enterprise/gitops-workflow.md:19]: Might flip this to say that the GitOps workflow is not supported for releases that use the Native Helm chart installation method.

(My understanding is that if you are packaging with a Helm chart and you select "Native" as the installation method, then your users can't use the GitOps workflow in the admin console. Is that what you take from it as well?)

Comment (jonquil2002) [reply docs/enterprise/gitops-workflow.md]: Yes, that's my understanding as well. Changed it.

**Comment by jonquil2002** [comment]
@paigecalvert Please review the most recent changes and approve if possible. Thx.

**Review by paigecalvert — APPROVED** [review]
LGTM

**Pull request merged**